templates/redis/bin/common.sh

```
@@ -13,16 +13,35 @@ POD_FQDN=$HOSTNAME.$SVC_FQDN
 # Extract pod IP from /etc/hosts
 POD_IP=$(grep "$HOSTNAME" /etc/hosts | awk '{print $1}' | head -1)
 
+# Detect redis-cli or fallback to valkey-cli
+if command -v redis-cli >/dev/null 2>&1; then
+    CLI_BIN="redis-cli"
+    SERVER_BIN="redis-server"
+    SENTINEL_BIN="redis-sentinel"
+elif command -v valkey-cli >/dev/null 2>&1; then
+    CLI_BIN="valkey-cli"
+    SERVER_BIN="valkey-server"
+    SENTINEL_BIN="valkey-sentinel"
```

Shellcheck (0.11.0)

[warning] SENTINEL_BIN appears unused. Verify use (or export if used externally).

(SC2034)